docs/src/pages/system/basics/basics.md

```diff
@@ -1,29 +1,19 @@
-# @material-ui/system
+# Material-UI System
 
-<p class="description">Styled system & style functions for building powerful design systems.</p>
+<p class="description">Utility style functions for rapidly building custom design systems.</p>
 
-## Getting Started
+Material-UI comes with dozens or **ready-to-use** components in the core.
+These components are an incredible starting point but when it comes to make your site stand out with a custom design, they can be challenging to customize. Introducing the system:
 
-`@material-ui/system` provides low-level utility functions called "_style functions_" for building powerful design systems. Some of the key features:
+The **system** lets you quickly build custom UI components leveraging the design tokens defined in your theme.
```

Comment:
I don't understand the logic.

Reply:
You wrote it. 😆

Reply:
Maybe it should be?

> Material-UI has dozens of **ready-to-use** (already styled) components. These components are an incredible starting point but when it comes to making your site stand out with a custom design, it can be simpler to start from an unstyled version. Introducing the system: